src/utils/populateDatabase.ts

```diff
@@ -0,0 +1,35 @@
+import dotenv from 'dotenv';
+import * as path from 'path';
+import * as fs from 'fs';
+import { AppSupabaseClient, Table } from '@/types';
+import { createClient } from '@supabase/supabase-js';
+import { Database } from '../lib/database.types';
+
+dotenv.config({ path: path.resolve(__dirname, '../..', '.env.local') });
+
+type EmbeddingDTO = Database['public']['Tables']['embedding']['Insert'];
+
+const entries: EmbeddingDTO[] = JSON.parse(
+  fs.readFileSync('./src/utils/data/dummyEmbeddings.json', 'utf8')
+);
+
+const supabase = createClient(
+  process.env.NEXT_PUBLIC_SUPABASE_URL ?? '',
+  process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY ?? ''
+);
+
+const insertItems = async (
```

Comment:
could call this `populateDatabase` to match file name (or rename file)

Reply:
true true